File: 800/1030A_InSearchOfAnEasyProblem.java
import java.io.*;
import java.util.*;

class Solution {
	public static void main (String args[]) {
		Scanner sc = new Scanner (System.in);
		int n = sc.nextInt();
		while (n -- != 0) {
			if (sc.nextInt() == 1) {
				System.out.println("HARD");
				return;
			}
		}
		System.out.println("EASY");
		return;
	}
}

File: 800/cpp/1030A_InSearchOfAnEasyProblem.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
	set<int> u;
	int n, i;
	cin >> n;
	while (n--) {
		cin >> i;
		if (i == 1) {
			cout << "HARD\n";
			return 0;
		}
	}
	cout << "EASY\n";
	return 0;
}